app/views/cards/_card_form.html.erb

```diff
@@ -0,0 +1,105 @@
+          <textarea class="form-control" rows="6" placeholder="Flashcard prompt ..."></textarea>
+        </div>
+        <div class="form-group">
+          <label>Answer</label>
+          <textarea class="form-control" rows="4" placeholder="Flashcard answer ..."></textarea>
+        </div>
+      </div>
+      <div class="tab-pane" id="shortAnswer">
+        <div class="form-group">
+          <label>Prompt</label>
+          <textarea class="form-control" rows="6" placeholder="Short Answer prompt ..."></textarea>
+        </div>
+        <div class="form-group">
+          <label>Answer</label>
+          <textarea class="form-control" rows="4" placeholder="Short Answer answer ..."></textarea>
+        </div>
+      </div>
+      <div class="tab-pane" id="organize">
+        <div class="form-group">
+          <label>Add Prompts</label>
+          <textarea class="form-control organize-prompt-text" rows="3" placeholder="New organize prompt ..."></textarea>
+          <div class="btn btn-default organize-prompt-btn-add">Add Prompt</div>
+        </div>
+        <label>Answer</label>
+        <ul id="organizePromptList" class="list-group organize-prompt-list">
+          <li class="list-group-item">
+            <span class="glyphicon glyphicon-resize-vertical"></span>
+            This is a sortable organize prompt list.</a>
+            <span class="glyphicon glyphicon-remove organize-prompt-btn-remove"></span>
+          </li>
+          <li class="list-group-item">
+            <span class="glyphicon glyphicon-resize-vertical"></span>
+            Add or remove items ...
+            <span class="glyphicon glyphicon-remove organize-prompt-btn-remove"></span>
+            </li>
+          <li class="list-group-item">
+            <span class="glyphicon glyphicon-resize-vertical"></span>
+            and drag them into the correct order.
+            <span class="glyphicon glyphicon-remove organize-prompt-btn-remove"></span>
+          </li>
+        </ul>
+        <p class="help-block organize-help-block">Add prompts using the input above.<br>Use the arrows to drag and drop prompts in their correct order</p>
+      </div>
+      <div class="tab-pane" id="fillInBlank">
+        <div class="form-group">
+          <label>Prompt</label>
+          <textarea class="form-control" rows="8" placeholder="Fill in the blank prompt ..."></textarea>
+          <p class="help-block">'One #two# three' will result in 'One ___ three'<br><%= link_to "https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet", target: "blank" %></p>
+        </div>
+      </div>
+    </div>
+  </div>
+  
+  <%= f.submit new ? 'Create' : 'Update', class: 'btn btn-brand' %>
+<% end %>
+
+<!-- Use https://github.com/RubaXa/Sortable for Drag + Drop support -->
+<script src="http://rubaxa.github.io/Sortable/Sortable.js"></script>
+
+<script>
+  Sortable.create(organizePromptList, {
+    handle: '.glyphicon-resize-vertical',
+    animation: 150
+  });
+</script>
```

Comment:
we'll need to coordinate on how `sortable` represents the order and what it will actually be saving to the DB